### PR TITLE
chore(portal): Fix flaky tests

### LINKED ISSUE #4509: Flaky Elixir unit test
https://github.com/firezone/firezone/actions/runs/8557001360/job/23448185850

### PR DESCRIPTION
**Pull request by AndrewDryga**
Closes #4509

### CODE REVIEW

File: elixir/apps/web/test/support/conn_case.ex [hunk 1]
```diff
@@ -233,4 +233,37 @@ defmodule Web.ConnCase do
...
+
+  @doc """
+  Waits for an ExUnit assertion to be `true` before timing out.
+
+  This is helpful when we check UI state changes in LiveView tests,
+  where we don't know if the view was updated before or after the test.
+
+  Default wait time is 2 seconds.
+  """
+  def wait_for(assertion_callback, wait_seconds \\ nil, started_at \\ nil) do
```
Comment (bmanifold) [inline elixir/apps/web/test/support/conn_case.ex]: This isn't blocking, just a style question.  Just wondering why the default value for `wait_seconds` isn't defined here, but rather in the rescue block?  Is that just a personal preference or is there an advantage to doing it this way?

**Review by bmanifold — APPROVED** [review]
LGTM

File: elixir/apps/web/test/support/conn_case.ex (hunk 1 above)
Comment (AndrewDryga) [reply elixir/apps/web/test/support/conn_case.ex]: @bmanifold good job spotting it, I just copied this code from acceptance tests but having it in the function body would be much better. I've pushed a fix for it in both places.

Comment (bmanifold) [reply elixir/apps/web/test/support/conn_case.ex]: 👍

**Pull request merged**